scripts/open-coredump.sh

```diff
@@ -341,4 +341,11 @@ Good luck!
 EOF
 fi
 
-exec ${SCYLLA_REPO_PATH}/tools/toolchain/dbuild -it -v $(pwd):/workdir -v ${SCYLLA_REPO_PATH}:/src/scylla -v ${ARTIFACT_DIR}/scylla.package:/opt/scylladb -w /workdir -- bash -l
+TOP_SRCDIR=$(dirname $(dirname $0))
```

Comment:
`$(dirname .)` == `.`, so this line will malfunction if you do `cd scripts; ./open_coredump.sh`. (`TOP_SRCDIR` will point to `scripts`).

Also it will break if the path contains spaces.

Also it doesn't respect symlinks.

How about `TOP_SRCDIR="$(dirname "$(realpath "$0")")"/..`?

Reply:
An alternative is to obtain `dbuild` from `master`/`enterprise`, just like we do with `scylla-gdb.py`, see above.
But using the one from the same repo from which `open-coredump.sh` is invoked from is also reasonable.

Reply:
thank you, Michał! indeed. i'd prefer not have `..` in the path even if the net result is identical. so i am keeping the two `dirname`s and switched to `readlink`.